support/mock/mockrt.go

```diff
@@ -97,6 +98,12 @@ type expectAggregateVerifySeals struct {
 	err     error
 }
 
+type expectReplicaVerify struct {
+	inSVIs  []proof.ReplicaUpdateInfo
```

Comment:
nit: SVI stands for seal verify info so you can change this field name to better match the new data in it.